### PR TITLE
Tg feature loginsignup

### PR DESCRIPTION
**Pull request by talgrewal**
# Feature Login Signup

- Folder Structure for login and signup.

### CODE REVIEW

File: client/js/App.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import {View, Text, StatusBar} from 'react-native';
+import Login from './screens/Login';
+
+export default class App extends Component {
+  render() {
+    return (
+      <>
+        {/* <StatusBar barStyle="dark-content"> */}
+        <Login />
+        {/* </StatusBar> */}
```
Comment (siddharthparmar7) [inline client/js/App.js]: `StatusBar` is a self-closing tag. Remove the comments if not needed.

File: client/js/components/AccountForm/AccountForm.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React, {Component} from 'react';
+import styles from './styles';
+import {Form, Field} from 'react-final-form';
+import {Text, View} from 'react-native';
+
+class AccountForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      formToggle: true,
```
Comment (siddharthparmar7) [inline client/js/components/AccountForm/AccountForm.js]: Create 2 screens, one for login and one for sign up. No need to toggle.

File: client/js/components/AccountForm/AccountForm.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      error: null,
+    };
+  }
+
+  render() {
+    return (
+      <View>
+        <Text>Login here</Text>
+        <Text>Username: ________</Text>
+        <Text>Password: ________</Text>
```
Comment (siddharthparmar7) [inline client/js/components/AccountForm/AccountForm.js]: Use final-form lib to create the login and sign up form.

File: client/js/screens/Login/Login.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import AccountFrom from '../../components/AccountForm';
+import styles from './styles';
+
+const Login = () => {
+  return <AccountFrom />;
```
Comment (siddharthparmar7) [inline client/js/screens/Login/Login.js]: This component should just render the final form for Login.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the requested changes. Thank you.

**Review by iyuvraajsingh — APPROVED** [review]

**Pull request merged**